src/core/ui.js

```diff
@@ -131,6 +133,35 @@ function createWarnButton(butName, arr, title) {
   return button;
 }
 
+function focusTrapper(e, firstFocussable, lastFocussable) {
+  if (e.key === "Tab") {
+    if (e.target == firstFocussable && e.shiftKey) {
+      e.preventDefault();
+      lastFocussable.focus();
+    } else if (e.target == lastFocussable && !e.shiftKey) {
+      e.preventDefault();
+      firstFocussable.focus();
+    }
+  }
+}
+
+menu.addEventListener("keydown", e => {
```

Comment:
We shouldn't need any of this, I don't think.

Reply:
Escaping is needed because trapping focus otherwise won't allow to turn the menu off with keyboard buttons.

Reply:
For a real field example, check out what 9-square button do in https://google.com.